opcodes.json

```diff
@@ -1306,6 +1328,25 @@
     "input": "address",
     "output": "",
     "description": "Halt execution and register account for later deletion",
-    "dynamicFee": true
+    "dynamicFee": {
+      "inputs": {
+        "callValue": {
+          "type": "number",
+          "label": "Call value"
+        },
+        "address": {
+          "type": "boolean",
```

Comment:
Same comment here, would prefer not exposing a boolean (the only boolean we need is the one below)